lib/active_storage/service/postgresql_service.rb

```diff
@@ -7,22 +7,22 @@ def initialize(public: false, **options)
     end
 
     def upload(key, io, checksum: nil, **)
-      instrument :upload, key: key, checksum: checksum do
-        ActiveStorage::Postgresql::File.create!(key: key, io: io, checksum: checksum)
+      instrument :upload, key:, checksum: do
+        ActiveStorage::Postgresql::File.create!(key:, io:, checksum:)
       end
     end
 
     def download(key)
       if block_given?
-        instrument :streaming_download, key: key do
+        instrument :streaming_download, key: do
           ActiveStorage::Postgresql::File.open(key) do |file|
             while data = file.read(5.megabytes)
               yield data
             end
           end
         end
       else
-        instrument :download, key: key do
+        instrument :download, key: do
           ActiveStorage::Postgresql::File.open(key) do |file|
```

Comment:
wow, it actually surprises me the ruby parser can parse this properly. i would have assumed this would need parens